docs/io-app/view-message-list.puml

```diff
@@ -0,0 +1,54 @@
+@startuml name ViewMessageList-ioApp
+participant MessagesHomeScreen
+participant PagerViewContainer
+participant MessageList
+participant Redux
+participant allPaginatedReducer
+participant HandleReloadAllMessages
```

Comment:
tip: you can use different `types` to make these actors more explicit (https://plantuml.com/sequence-diagram#:~:text=authentication%20Response%0A%40enduml-,Declaring%20participant,-If%20the%20keyword)

Reply:
Thanks! I've customised Redux and BE